### PR TITLE
Revised ROSGeometry docs

### PR DESCRIPTION
**Pull request by LaurieCheers-unity**
https://jira.unity3d.com/browse/AIRO-667

### CODE REVIEW

File: ROSGeometry.md [hunk 1]
```diff
@@ -2,12 +2,48 @@
...
 
-The ROSGeometry namespace contains code to make it easier to work with these various coordinate frames - letting you be explicit about what coordinates a given value is in at compile time, and managing the conversions for you. It does this with two generic structs, `Vector3<C>` and `Quaternion<C>`. The type parameter C indicates the coordinate frame you're working in - either FLU, or RUF, or perhaps a more exotic frame such as NED (north, east, down) or ENU (east, north, up), commonly used in aviation.
+The ROSGeometry namespace contains code to make it easier to work with these various coordinate frames - letting you be explicit about what coordinates a given value is in at compile time, and managing the conversions for you.
 
+# Ros Message conversions:
+
+The main ROS position messages (geometry_msgs/Point, geometry_msgs/Point32 and geometry_msgs/Vector3) can be converted to and from Unity Vector3s like this:
+
+	PointMsg myRosPoint = transform.position.To<FLU>();
```
Comment (sdiao) [inline ROSGeometry.md:11]: what is a `transform`? Do we need to add a variable definition?

Comment (sdiao) [reply ROSGeometry.md:11]: just reading through the tutorial and my understanding is that each game object has a transform. I wonder if we want to use something like `myObject.trasnform.position.To<FLU>()` in the examples.

**Review by sdiao — APPROVED** [review]

File: ROSGeometry.md (hunk 1 above)
Comment (LaurieCheers-unity) [reply ROSGeometry.md:11]: Oh, yeah, I guess we could provide more context for people with less Unity experience.
Usually in Unity scripting the code you're writing lives in a subclass of MonoBehaviour, which has a property called transform (which gets the transform of its gameobject).

**Review by sdiao — APPROVED** [review]
lgtm

**Pull request merged**